src/Alice/Loader/Neon.php

```diff
@@ -70,7 +70,7 @@ private function processIncludes($data, $file)
 			foreach ($data['includes'] as $include) {
 				$includeFile = dirname($file) . DIRECTORY_SEPARATOR . $include;
 				$includeData = $this->parse($includeFile);
-				$data = Helpers::merge($data, $includeData);
+				$data = Helpers::merge($includeData, $data);
```

Comment:
Because all included files were processed in reversed order.

Eg, if you have defined

    includes:
        - user1.neon
        - user2.neon

It firstly added `user2` and then `user1`, which was kind of non-intuitive.